Makefile

```diff
@@ -17,7 +19,9 @@ lint:
 	docker-compose -f docker-compose.test.yml run --rm tests gosec ./...
 
 smoke-test:
-	docker-compose -f docker-compose.test.yml run --rm -w /go/src/github.com/CMSgov/bcda-app/test/smoke_test tests sh smoke_test.sh
+	docker exec -it bcda-app_api_1 sh -c 'tmp/bcda create-alpha-token --size dev' > /tmp/creds
```

Comment:
I think we should try to avoid saving the output of `create-alpha-token` to a temp file.  Instead, maybe we can leverage some nifty commands like `cut` to save the output of the commands into vars?